<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgets::GeomControllersWidget</class>
 <widget class="QWidget" name="widgets::GeomControllersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="controllerSelector"/>
     </item>
     <item>
      <widget class="QPushButton" name="editControllerListButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="controllerProperties"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
